RubeesEat.IntegrationTests/PageObjects/PersonButtonControlObject.cs

```diff
@@ -0,0 +1,16 @@
+using OpenQA.Selenium;
+using RubeesEat.IntegrationTests.WebTesting;
+
+namespace RubeesEat.IntegrationTests.PageObjects;
+
+public class PersonButtonControlObject : ControlObject
+{
+    private readonly PageObject _page;
+
+    public PersonButtonControlObject(PageObject page, IWebElement me) : base(page, me)
+    {
+        _page = page;
+    }
```

Comment:
We don't need to save the `PageObject` in a field as it is already done by the base class.